ffi/gn/icu4x/BUILD.gn

```diff
@@ -37,6 +37,53 @@ rust_library("calendrical_calculations-v0_1_0") {
   visibility = [ ":*" ]
 }
 
+rust_library("databake-v0_1_5") {
```

Comment:
databake should be a datagen-only dependency and not be pulled in here

Reply:
Indeed